Update README

miguelangel-dev opened the pull request:
### Description
In this PR, I have updated the README to keep sync with the current nef version (after refactor). Also, I have fixed minor bugs.

miguelangel-dev committed:
update installing, playground, compiling, clean sections to new nef version

miguelangel-dev committed:
fix bug in extract platform from a playground page with invalid characters

miguelangel-dev committed:
Keep aligned the README documentation with the nef commands

miguelangel-dev committed:
remove unuseful assets

miguelangel-dev committed:
fix bug setting dependencies for cocoapods

minor changes

miguelangel-dev committed:
update badge status

Update README.md

miguelangel-dev committed:
information about how to install nef - readme

truizlop commented on README.md:
````diff
@@ -54,19 +54,21 @@
 .package(url: "https://github.com/bow-swift/nef.git", from: "{version}")
 ```
 
+It is an excellent option if you want to use the full nef features in your macOS app, even to build new tooling on top of nef.
````
```suggestion
It is an excellent option if you want to use all nef features in your macOS app, even to build new tooling on top of nef.
```

truizlop commented on README.md:
```diff
@@ -54,19 +54,21 @@
 #### 🔌 Using [Xcode Editor Extension](https://github.com/bow-swift/nef-plugin)
 
-Some of the `nef` features can be used directly in Xcode as an Extension. You can install it directly from [**App Store**](https://apps.apple.com/app/nef/id1479391704?mt=8).
+Some of the `nef` features can be used directly in Xcode as an Extension. You can install it directly from [**App Store**](https://apps.apple.com/app/nef/id1479391704?mt=8) or downloading the last binary from the [**releases section**](https://github.com/bow-swift/nef-plugin/releases).
```
```suggestion
Some of `nef` features can be used directly in Xcode as an Extension. You can install it directly from [**App Store**](https://apps.apple.com/app/nef/id1479391704?mt=8) or downloading the last binary from the [**releases section**](https://github.com/bow-swift/nef-plugin/releases).
```

truizlop commented on README.md:
```diff
@@ -80,24 +82,32 @@ By default, `nef` can create an Xcode Playground with support for [Bow](http://b
 
 <details>
-<summary>📣 You can select any different Bow version or branch, even a third-party dependency</summary>
+<summary>📣 You can create a nef Playground compatible to any different Bow version, branch or commit; even third-parties dependencies</summary>
```
```suggestion
<summary>📣 You can create a nef Playground compatible with any different Bow version, branch or commit; even third-party dependencies</summary>
```

truizlop commented on README.md:
```diff
@@ -156,26 +174,27 @@ github "bow-swift/Bow"
+### 🔨 Compiling a nef Playground
 
-Xcode lets you check for correctness of your Xcode Playground and run it. However, compiling an Xcode Playground from the command line is not so easy when it has dependencies on third-party libraries. This is particularly useful in Continuous Integration when you want to verify that your playgrounds are not broken when the libraries you depend on are updated. `nef` has an option to compile Xcode  ...
+Xcode lets you check for correctness of your Xcode Playground and run it. However, Apple does not give us compiling commands for Xcode Playground as they do for building Xcode projects. It is particularly useful in Continuous Integration when you want to verify that your playgrounds are not broken when the libraries you depend on are updated. `nef` has an option to compile a `nef Playground`. To  ...
```
```suggestion
Xcode lets you check for correctness of your Xcode Playground and run it. However, Apple does not provide us commands to compile an Xcode Playground, as they do for building Xcode projects. It is particularly useful in Continuous Integration when you want to verify that your playgrounds are not broken when the libraries you depend on are updated. `nef` has an option to compile a `nef Playground`. To do this, you can run the following command:
```

miguelangel-dev committed:
Update README.md

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update README.md

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update README.md

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

Update README.md

Co-Authored-By: Tomás Ruiz-López <truizlop@gmail.com>

minor changes about project name in README
